src/StorageWriter/DatabaseMongo.ts

```diff
@@ -0,0 +1,66 @@
+import { Claim } from '@po.et/poet-js'
+import { inject, injectable } from 'inversify'
+import { Collection, Db, FindAndModifyWriteOpResultObject } from 'mongodb'
+import { isNil, pipeP, lensPath, view } from 'ramda'
+
+import { Database } from './Database'
+import { DatabaseMongoConfiguration } from './DatabaseMongoConfiguration'
+
+const L = {
+  valueClaim: lensPath(['value', 'claim']),
+}
+
+export const getClaimFromFindAndUpdateResponse = (response: FindAndModifyWriteOpResultObject): Claim | undefined =>
+  view(L.valueClaim, response)
+
+export const throwIfClaimNotFound = (claim: Claim): Claim => {
```

Comment:
@lautarodragan are you good with how I've removed this function from the class so it could be unit tested?